src/userConfig_sample.h

```diff
@@ -28,16 +28,27 @@ enum MACHINE {
 // Machine
 #define MACHINEID RancilioSilvia   // Machine type (see enum MACHINE)
 
+/* Features
+ *
+ * 0 = deactivated, 1 = activated
+ */
+#define FEATURE_HEATINGLOGO 1
```

Comment:
I like the rename but wouldn't it be nice to keep the on/off flag next to the options that relate to that feature?

Reply:
It seemed like a good idea to have the features all in one place but I'm not married to the idea :)